src/app/components/header/header.component.ts

```diff
@@ -7,4 +8,12 @@ import { Component } from '@angular/core';
 })
 export class HeaderComponent {
 
+  faBars = faBars;
+
+  toggleDropdown(): void {
```

Comment:
Can you please check this component in Angular Material?

https://material.angular.io/components/menu/overview

Reply:
Can you please also rebase you branch with `main`? There were some changes in the header file.

Reply:
Actually I will unblock this PR, the header must change and this is not the scope of this story.